rdkcentral/Dobby
# Fix DobbyTool Bundle segfault

### Description
If the `DobbyTool bundle` command is accidentally provided a directory as an argument instead of a file, DobbyTool will segfault. Check if arg is a file and return an error if not.

### Test Procedure
Start DobbyDaemon. Run `DobbyTool bundle test /tmp`. Should return an error instead of segfaulting. 

### Type of Change

### Requires Bitbake Recipe changes?
